app/models/user.rb

```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly add `confirmable`, as it is a requirement that the the user email must be confirmable when a user is signing up.

Reply:
Thank you for the review and feedback @Munsa1. I fixed it now.👍